server/server.js

```diff
@@ -11,3 +11,24 @@
 // iterating through that list, display the string you built up.
 
 // Remember that the response is a stream object that must be closed.
+
+
+var http = require('http');
+var enhancedDate = require('./enhancedDate');
+var filterByDay = require('./trucks');
+
+http.createServer(function (request, response) {
+	var currentDay = enhancedDate.getDayName();
+	var trucks = filterByDay(currentDay);
+	var output;
```

Comment:
When you declare a variable that you're going to use to build up a string, initialize it to an empty string. Otherwise, when you start concatenating other strings onto it, the string will begin with "undefined" as nothing was assigned to it before additional strings were concatenated onto it.